AppCenterDataStorage/AppCenterDataStorage/Internal/Client/MSTokenExchange.h

```diff
@@ -0,0 +1,21 @@
+#import <Foundation/Foundation.h>
+#import "MSHttpIngestion.h"
```

Comment:
Can be replaced with just forward declaration

Reply:
It isn't resolved

Reply:
what's the problem with adding the header?

Reply:
Using forward declaration is a best practice in general but It particularly helps in getting rid of circular dependencies like A imports B and B imports A. Instead of `#import "MSHttpIngestion.h"` you would use `@class MSHttpIngestion;`. You would do that on classes and protocols the class you are declaring is not inheriting from.